lib/python_prefida/write_data.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+
+def write_data(h5_obj, dic, desc, units, name=''):
+    #+##`write_data(h5_obj, dic, desc, units)`
+    #+ Write h5 datasets with attributes 'description' and 'units'
+    #+###Arguments
+    #+     **h5_obj**: An h5 file or group object from h5py
+    #+
+    #+     **dic**: Dict of data to save as h5 datasets
+    #+
+    #+     **desc**: Dict with same keys as dic describing each item in dic
+    #+
+    #+     **units**: Dict with same keys as dic providing units of data in dic, doesn't have to be all keys of dic.
+    #+
+    #+###Keyword Arguments
+    #+     **name**: Name/description of dic for clarity in raising errors
+    #+
+    #+###Example Usage
+    #+```python
+    #+>>> uvw_to_xyz(h5_obj, dic, desc, units)
```

Comment:
It should be `write_data(h5_obj, dic, desc, units)` not `uvw_to_xyz(h5_obj, dic, desc, units)`